compiler/rustc_trait_selection/src/traits/wf.rs

```diff
@@ -75,6 +75,30 @@ pub fn obligations<'tcx>(
     Some(result)
 }
 
+pub fn unnormalized_obligations<'tcx>(
```

Comment:
please add a comment that this should only be used in the new solver as it does not track span information.